build/dockerfiles/coordinator-api.Dockerfile

```
@@ -1,31 +1,25 @@
 # Build libzkp dependency
-FROM scrolltech/cuda-go-rust-builder:cuda-11.7.1-go-1.22.12-rust-nightly-2025-02-14 as chef
+FROM scrolltech/go-1.22.12-rust-nightly-2025-02-14 as chef
 WORKDIR app
```

Hadolint (2.12.0)

[error] Use absolute WORKDIR

(DL3000)